setup.cfg

```diff
@@ -82,11 +82,73 @@ setup_requires =
     wheel
 zip_safe = False
 
+[options.packages.find]
+exclude =
+    build*
+    dist*
+    docs*
+    scripts*
+    typings*
+    hooks*
+    .git*
+    tests*
+    *.tests
+    *.tests.*
+    *.yml
```

Comment:
don't we have `.yml` file as part of the distribution? the default configuration file

Reply:
will just remove this part then. This is to accomodate upstream changes from setuptools